pkg/expect/expect.go

```diff
@@ -32,19 +33,31 @@ type ExpectProcess struct {
 	fpty *os.File
 	wg   sync.WaitGroup
 
-	ptyMu sync.Mutex // protects accessing fpty
-	cond  *sync.Cond // for broadcasting updates are available
-	mu    sync.Mutex // protects lines and err
-	lines []string
-	count int // increment whenever new line gets added
-	err   error
+	ptyMu          sync.Mutex // protects accessing fpty
+	cond           *sync.Cond // for broadcasting updates are available
+	mu             sync.Mutex // protects lines and err
+	lines          []string
+	count          int // increment whenever new line gets added
+	err            error
+	ShutdownSignal os.Signal
```

Comment:
`StopSignal` since it's the signal sent by calling `Stop`? Also needs a godoc since it's public:
```go
// StopSignal is the signal Stop sends to the process; defaults to SIGKILL.
StopSignal os.Signal
```